js/web/lib/wasm/jsep/webgpu/ops/instance-norm.ts

```diff
@@ -207,7 +207,7 @@ const computeMean =
     let offset = currentImageNumber * uniforms.image_size;
     var sum = ${fillVector('f32', components)};
     var squaredSum = ${fillVector('f32', components)};
-    for (var i: u32 = 0; i < ${WG}; i++) {
+    for (var i: u32 = 0; i < min(${WG}, uniforms.H); i++) {
```

Comment:
I don't fully understand the algorithm, but it seems to me this fix doesn't work if uniforms.H is larger than 64. All the test cases pass because all the Hs there are smaller than 64?

Reply:
The mean is computed in two steps. 1), InstanceNormComputeMean 2), InstanceNormComputeChannelScaleShift.
When H gets bigger(compared to fixed WG), more calculation happens in InstanceNormComputeMean, less in InstanceNormComputeChannelScaleShift. So I think the result will be OK.

Reply:
After further chat with Xing, I'm convinced his change is correct. Only if uniforms.H is smaller than WG, we may have uninitialized values in 64 buckets.